waiter/resources/web/welcome.html

```diff
@@ -0,0 +1,165 @@
+     font-size: 1.2rem;
+     margin-left: 1em;
+   }
+
+   .divider {
+     /* ui divider */
+     border-top: 1px solid rgba(34,36,38,.15);
+     border-bottom: 1px solid rgba(255,255,255,.1);
+     margin: 1rem 0;
+     line-height: 1;
+     height: 0;
+     font-weight: 700;
+     text-transform: uppercase;
+     letter-spacing: .05em;
+     color: rgba(0,0,0,.85);
+   }
+
+   .additional-info {
+     width: 800px;
+     margin: auto;
+     margin-top: 2em;
+   }
+
+   .additional-info p {
+     margin-left: 1em;
+   }
+
+   .additional-info h3 {
+     margin-top: 2em;
+   }
+
+   .additional-info li {
+     line-height: 2em;
+   }
+
+   .code {
+     font-family: "Lucida Console", Monaco, monospace;
+     line-height: 2em;
+     font-size: 0.9rem;
+   }
+
+   pre.code {
+     margin-left: 1em;
+     margin-right: 1em;
+     white-space: pre-wrap;
+     font-size: 0.8rem;
+   }
+
+   .additional-info td {
+     padding-right: 1em;
+   }
+
+   .additional-info td.field {
+     font-weight: bold;
+     text-align: right;
+     vertical-align: baseline;
+     width: 70px;
+   }
+
+   .additional-info td.code {
+     vertical-align: baseline;
+   }
+
+   .additional-info table {
+     margin-left: 1em;
+     border-collapse: collapse;
+   }
+    </style>
+</head>
+<body>
+
+<div class="main-content">
+  <h1>
+    Welcome to Waiter
+  </h1>
+  <p>
+    Waiter is a high-level platform for running and managing web services and applications.
+  </p>
+  <p>
+    See <a href="#gettinghelp">Getting Help</a> for more info, or if you feel you've reached this page in error.
+  </p>
+</div>
+
+<div class="additional-info">
+  <div class="divider"></div>
+  <div>
+    <h3>Server Info</h3>
+    <table>
+      <tr>
+        <td class="field">Host</td>
+        <td class="code"><%= hostname %></td>
+      </tr>
+      <tr>
+        <td class="field">Interface</td>
+        <td class="code"><%= host %></td>
+      </tr>
+      <tr>
+        <td class="field">Port</td>
+        <td class="code"><%= port %></td>
+      </tr>
+    </table>
+    <p>
+      <a href="/settings">More Settings</a>
+    </p>
+  </div>
+</div>
+
+<div class="additional-info">
+  <div class="divider"></div>
+  <div>
+    <h3>Request Info</h3>
+    <table>
+      <tr>
+        <td class="field">CID</td>
+        <td class="code"><%= cid %></td>
+      </tr>
+      <tr>
+        <td class="field">Time</td>
+        <td class="code"><%= timestamp %></td>
+      </tr>
+    </table>
+  </div>
+</div>
+
+
+<div class="additional-info" id="gettinghelp">
+  <div class="divider"></div>
+  <div>
+    <h3>Getting Help</h3>
+    <ul>
+      <% (doseq [{label :label {:keys [type value]} :link} support-info] %>
+      <li>
+        <%= label %>:
+        <% (when (= type :url) %> <a href="<%= value %>"><%= value %></a><% ) %>
+        <% (when (= type :email) %> <a href="mailto:<%= value %>"><%= value %></a><% ) %>
+      </li>
+      <% ) %>
+    </ul>
+</div>
+
+</body>
+
+</html>
```

Comment:
I wanted to keep it brief, directing people to appropriate links on the bottom.  We can easily add a Getting Started link at the bottom.

If we add this, I'm not sure what else isn't appropriate for the welcome page.

Reply:
More thoughts:
 - passing `x-waiter-token` isn't limited to Javascript
 - passing additional headers is also an option

Adding all the details on what the all the valid ways to create a service is too much for a welcome page, and only mentioning some of the options seems confusing to me.

I'm open to a rewording.

Reply:
I agree that it could be too much for a welcome page. How about linking to a built-in Getting Started page that's also served by Waiter and contains this information?

Reply:
- Doesn't that create another documentation venue that we'd have to worry about keeping consistent?
- How is that better than a link at the bottom of the page that could contain the same information?
- How would you allow installations that customize Waiter's functionality to change the documentation to suit their customized semantics?

Reply:
Sounds good, I'm fine with that